test/unit/extension/lib/mergeCartsOnLoginTest.js

```diff
@@ -111,19 +111,43 @@ describe('mergeCartsOnLogin', () => {
       }
     }
 
-    const pipelineStub = sandbox.createStubInstance(ShopgateCartPipeline)
```

Comment:
I know you just did some changes but could you please also add:

const {describe, it, beforeEach, afterEach} = require('mocha')

and run npm i @types/mocha @types/sinon @types/bunyan @types/chai @types/chai-as-promised -D

Reply:
Done - npm i was not necessary.